sdk/core/core-lro/package.json

```diff
@@ -78,9 +78,9 @@
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint -c ../../../.eslintrc.json package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint -c ../../../.eslintrc.json package.json api-extractor.json src test  --ext .ts",
-    "lint:terminal": "eslint -c ../../../.eslintrc.json package.json api-extractor.json src test --ext .ts",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test  --ext .ts",
+    "lint:terminal": "eslint package.json api-extractor.json src test --ext .ts",
```

Comment:
We are not using `lint:terminal` anywhere anymore. It was put in place at a time when we thought that it was needed because the `lint` script would print the output to the html file and having something print to the console was useful

```suggestion
```